README.md

```diff
@@ -1,3 +1,11 @@
+[![codecov](https://codecov.io/gh/opensearch-project/OpenSearch/branch/2.0/graph/badge.svg)](https://codecov.io/gh/opensearch-project/OpenSearch)
+[![GHA gradle check](https://github.com/opensearch-project/OpenSearch/actions/workflows/gradle-check.yml/badge.svg)](https://github.com/opensearch-project/OpenSearch/actions/workflows/gradle-check.yml)
+[![GHA validate pull request](https://github.com/opensearch-project/OpenSearch/actions/workflows/wrapper.yml/badge.svg)](https://github.com/opensearch-project/OpenSearch/actions/workflows/wrapper.yml)
+[![GHA precommit](https://github.com/opensearch-project/OpenSearch/actions/workflows/precommit.yml/badge.svg)](https://github.com/opensearch-project/OpenSearch/actions/workflows/precommit.yml)
+[![Jenkins gradle check job](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fbuild.ci.opensearch.org%2Fjob%2Fgradle-check%2F&label=Jenkins%20Gradle%20Check)](https://build.ci.opensearch.org/job/gradle-check/)
+[![Documentation](https://img.shields.io/badge/doc-reference-blue)](https://opensearch.org/docs/latest/opensearch/index/)
+[![Chat](https://img.shields.io/badge/chat-on%20forums-blue)](https://forum.opensearch.org/c/opensearch/)
+
```

Comment:
These should not be above the project logo.

If moved below the logo, may want to separate a few "informational" badges that we want up front (e.g., documentation and chat, and possibly add one for license, and possibly the latest release) from the laundry list of CI badges that will probably grow in the future and are of much less interest to visitors to the project.  They could be added to a CI Status section later on.

Compare my old project after badge explosion 
 https://github.com/oshi/oshi/blob/oshi-parent-6.1.0/README.md
vs the new badge setup:
 https://github.com/oshi/oshi/blob/master/README.md

Reply:
Thanks @dbwiddis for your review.

Moved all badges after logo and moved chat/doc in front.